src/UI/CompletionsView.re

```diff
@@ -68,192 +63,186 @@ let completionKindToIcon: option(Ext.CompletionItemKind.t) => int =
     |> Option.value(~default=FontAwesome.question);
   };
 
-let completionKindToColor =
+module Styles = {
+  open Style;
+
+  let outerPosition = (~x, ~y) => [
+    position(`Absolute),
+    top(y - 4),
+    left(x + 4),
+  ];
+
+  let innerPosition = (~height, ~width, ~lineHeight, ~theme: Theme.t) => [
+    position(`Absolute),
+    top(int_of_float(lineHeight +. 0.5)),
+    left(0),
+    Style.width(width),
+    Style.height(height),
+    border(~color=theme.editorSuggestWidgetBorder, ~width=1),
+    backgroundColor(theme.editorSuggestWidgetBackground),
+  ];
+
+  let item = (~isFocused, ~theme: Theme.t) => [
+    isFocused
+      ? backgroundColor(theme.editorSuggestWidgetSelectedBackground)
+      : backgroundColor(theme.editorSuggestWidgetBackground),
+    flexDirection(`Row),
+  ];
+
+  let icon = (~color) => [
+    flexDirection(`Row),
+    justifyContent(`Center),
+    alignItems(`Center),
+    flexGrow(0),
+    backgroundColor(color),
+    width(25),
+  ];
+
+  let label = [flexGrow(1), margin(4)];
+
+  let text =
+      (~highlighted=false, ~theme: Theme.t, ~editorFont: EditorFont.t, ()) => [
+    textOverflow(`Ellipsis),
+    fontFamily(editorFont.fontFile),
+    fontSize(editorFont.fontSize),
+    textWrap(TextWrapping.NoWrap),
+    Style.color(
+      highlighted ? theme.oniNormalModeBackground : theme.editorForeground,
+    ),
+  ];
+
+  let highlightedText = (~theme: Theme.t, ~editorFont: EditorFont.t) =>
+    text(~highlighted=true, ~theme, ~editorFont, ());
+
+  let detail = (~width, ~lineHeight, ~theme: Theme.t) => [
+    position(`Absolute),
+    left(width),
+    top(int_of_float(lineHeight +. 0.5)),
+    Style.width(Constants.maxDetailWidth),
+    flexDirection(`Column),
+    alignItems(`FlexStart),
+    justifyContent(`Center),
+    border(~color=theme.editorSuggestWidgetBorder, ~width=1),
+    backgroundColor(theme.editorSuggestWidgetBackground),
+  ];
+
+  let detailText = (~editorFont: EditorFont.t, ~tokenTheme: TokenTheme.t) => [
```

Comment:
Is the detail text still showing?

__`master`:__
![image](https://user-images.githubusercontent.com/13532591/71862210-f10f2b00-30ad-11ea-866e-016887d56f60.png)

__`pr`:__
![image](https://user-images.githubusercontent.com/13532591/71862138-b907e800-30ad-11ea-93e2-00df19b67f88.png)

Also - if you compare the screenshots - you can see the difference the gamma-correction makes in the text rendering. The un-gamma-corrected text in the PR makes the text appear thin and washed out.

Reply:
Will look into it.

Reply:
They're showing up for me. And even change when I move up and down the list. You don't seem to have any suggestion focused though. Not sure how you managed that.

Reply:
I see it now with the latest! It looks like this theme isn't showing the suggested highlight - but if I switch to another theme I can see it 🤔